internal/nozzle/nozzle.go

```diff
@@ -198,24 +199,30 @@ func (n *Nozzle) postMetrics() {
 	n.mapLock.Unlock()
 
 	timestamp := time.Now().Unix()
+
+	n.totalMetricsReceived += uint64(len(metricsMap))
+
 	for _, client := range n.ddClients {
 		// Add internal metrics
 		k, v := client.MakeInternalMetric("totalMessagesReceived", totalMessagesReceived, timestamp)
 		metricsMap[k] = v
+		k, v = client.MakeInternalMetric("totalMetricsReceived", n.totalMetricsReceived, timestamp)
+		metricsMap[k] = v
 		k, v = client.MakeInternalMetric("totalMetricsSent", n.totalMetricsSent, timestamp)
 		metricsMap[k] = v
```

Comment:
right now, we're sending those as gauges. It doesn't make much sense, we should instead be sending those as counts.
It would also make more sense to directly add a metric for the number of dropped metrics.
I'd suggest adding a metrics `metrics.dropped` which would be a count (reset it to zero every time we flush metrics), and a metric `metrics.sent`, a count as well, and we wouldn't need a `metric.received`